### DISCUSSION TITLE
Is this a good place to be holding discussions about things that are not yet fully-formed issues?

### DISCUSSION THREAD

**Discussion by geryan** [njtierney/pengweens#6] [Ideas]
E.g., before #3 is fully formed. 

Let's put a pin in it 🤘🏼

**Comment by njtierney** [comment]
We can also move between them - @smwindecker this might be a cool solution/bridge here

e.g.,

issues -- > discussions

<img width="1264" alt="image" src="https://github.com/njtierney/pengweens/assets/6488485/6dcdcb44-cb77-4863-a1e6-4d5220271a5c">


discussion -- > issues

<img width="1239" alt="image" src="https://github.com/njtierney/pengweens/assets/6488485/0532a04d-84f6-4b7c-b5f1-fba1b93c9cef">
